capstone/src/main/java/com/step902020/capstone/EventTemp.java

```diff
@@ -0,0 +1,26 @@
+package com.step902020.capstone;
+
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Entity;
+import org.springframework.cloud.gcp.data.datastore.core.mapping.Field;
+import org.springframework.data.annotation.Id;
+
+@Entity(name = "eventTemp")
+public class EventTemp {
```

Comment:
Haha I love the bare-bones functionality here, I'll be sure to get you something to actually work with soon!